HW1.java

```diff
@@ -0,0 +1,13 @@
+public class HW1 {
+
+        int Year = 2023;
```

Comment:
method should reflect the sense of it - please rename the method as sumOfTwoNumbers